tools/python/boutiques/bosh.py

```diff
@@ -436,6 +481,12 @@ def bosh_return(val, code=0, hide=False):
         elif func == "test":
             out = test(*params)
             return bosh_return(out)
+        elif func == "search":
+            out = search(*params)
+            return bosh_return(out)
+        elif func == "pull":
+            out = pull(*params)
+            return bosh_return(out)
```

Comment:
not sure if we want to print the value returned by pull when used in CLI. If not, set `hide` to `True`.

Reply:
Ok, but setting `hide=True` means it won't print `OK`. I guess that's not a big deal.

Reply:
It sounds good indeed.